app/models/transaction.rb

```diff
@@ -11,10 +11,12 @@ def amount=(value)
   private
 
   def return_amount_with_two_decimal(value)
```

Comment:
At first, I wanted to use this code, which working in browser ok
```ruby
def return_amount_with_two_decimal(value)
  value = value.to_f
  if value == 0
    errors.add(:amount, 'Write a number, please')
  else
    value.floor(2)
  end
end
```
but tests were failed, because of 0 (I cheked that the value is greater than 0)

So, I decided to use code bellow